### DISCUSSION TITLE
Tandy Models

### DISCUSSION THREAD

**Discussion by mfmcosta** [86Box/86Box#3809] [Ideas]
**What's up with them?**
[8088] Tandy 1000 - The BIOS is the v1.02.00 which belongs to the Tandy 1000EX (or Tandy 1000SX)
[8088] Tandy 1000HX - Supports VGA
[8086] Tandy 1000SL/2 - Supports VGA

**Where did you get that info?**
The info was obtained from the [Tech Notes](https://www.lo-tech.co.uk/downloads/manuals/tandy/Tandy_1000_Tech_Notes_and_Jumper_Manual-vol-1.pdf)
on page 8 the list of the BIOS version per model
on page 22:
> You can install VGA in all of the 1000's, except the 1000, 1000A and 1000HD. The built-in CGA in these systems cannot be disabled.

And inside the TANDY1T1.020 says:
![image](https://github.com/86Box/86Box/assets/136728257/e2ff69d1-9632-42cf-a371-fa0f485ea8b2)
And the filename says "1.020"

**Do you have a Tandy 1000 ROM?**
I've found none on internet but I've found the version 1.010 (for Tandy 1000A or Tandy 1000HD)
[TANDY1T1.ZIP](https://github.com/86Box/86Box/files/12524529/TANDY1T1.ZIP)

**We will see what can be done**
Thank you!

**Comment by OBattler** [comment]
I'd like to see some some hard evidence that they only used one single BIOS version of each machine. Even IBM didn't do that. So this argument by version sounds to me a bit like arguing that the IBM PC 1982 BIOS must be the XT BIOS because of the date.

**Comment by Digitoxin1** [comment]
**Tandy 1000 Bios revisions:**

- Tandy 1000 - 01.00.00 upgradable to 01.01.00 to support hard drive controllers
- Tandy 1000A - 01.01.00
- Tandy 1000HD - 01.01.00
- Tandy 1000SX - 01.02.00
- Tandy 1000EX - 01.02.00
- Tandy 1000TX - 01.03.00
- Tandy 1000HX - 02.00.00
- Tandy 1000SL - 02.00.00 upgradable to 02.00.01 for Deskmate v.3.05
- Tandy 1000SL/2 - 02.00.00 upgradable to 02.00.01 for Deskmate v.3.05
- Tandy 1000TL - 02.00.00 upgradable to 02.00.01 for Deskmate v.3.05
- Tandy 1000TL/2 - 02.00.00 upgradable to 02.00.01 for Deskmate v.3.05
- Tandy 1000TL/3 - 02.00.00 upgradable to 02.00.01 for Deskmate v.3.05

**Reply by pixel-jupiter** [reply]
According to the _Tandy 1000 Tech Notes and Jumper Settings Manual_, some models have different or additional BIOS versions compared to the list above:

- Tandy 1000SL - 01.04.00, 01.04.01, 01.04.02 or 02.00.01
- Tandy 1000SL/2 - 01.04.04 or 02.00.01
- Tandy 1000TL - 01.04.00, 01.04.01, 01.04.02 or 02.00.01

Additional entries from the same source:

- Tandy 1000RL - 02.00.00 or 02.00.01
- Tandy 1000RL-HD - 02.00.00 or 02.00.01
- Tandy 1000RLX - 02.00.00
- Tandy 1000RLX-HD - 02.00.00
- Tandy 1000RLX-B - 02.00.00
- Tandy 1000RLX-HD-B - 02.00.00
- Tandy 1000RSX - 01.10.00
- Tandy 1000RSX-HD - 01.10.00

**Comment by Fenix770** [comment]
I have some Tandy BIOS Files, I downloaded about 15 years ago.
(I think I downloaded from retrograde station site)

Tandy 1000 ROM v01.01.00 (19 kb)
Tandy 1000 ROM v01.02.00 (22 kb) 
Tandy 1000 ROM v01.03.00 (23 kb)
Tandy 1000HX ROM v02.00.00 (44 kb)
Tandy 1000HX ROM v02.00.00 for use in MESS (89 kb) (has DOS v2.11 in ROM)
Tandy 1000 TL/2 ROM v02.00.00 (19 kb) 
AT class Tandy ROM v02.00.00 (30 kb) 

[TANDY ROMS.zip](https://github.com/user-attachments/files/16276035/TANDY.ROMS.zip)
